scripts/config

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+source /usr/share/yunohost/helpers
+ynh_abort_if_errors
+
+CUSTOM_ADDONS=$final_path/custom-addons
+VENV=$final_path/venv
+CUSTOM_REPO=$CUSTOM_ADDONS/custom-repo
+REPOS_YML=$CUSTOM_REPO/repos.yml
+REQUIREMENTS_TXT=$CUSTOM_REPO/requirements.txt
+
+_setup_git() {
+  pushd $CUSTOM_ADDONS>/dev/null
+    ynh_exec_as $app git config --global user.email $app@$(hostname)
+    ynh_exec_as $app git config --global user.name $app
+    ynh_exec_as $app git config --global init.defaultBranch main
+    ynh_exec_as $app git config --global pull.rebase false
+  popd>/dev/null
```

Comment:
```suggestion
  pushd $CUSTOM_ADDONS
    ynh_exec_as $app git config --global user.email $app@$(hostname)
    ynh_exec_as $app git config --global user.name $app
    ynh_exec_as $app git config --global init.defaultBranch main
    ynh_exec_as $app git config --global pull.rebase false
  popd
```

kind of nitpicking but it feels a bit odd to redirect to >/dev/null, i guess it doesnt hurt, but at the same time it's unecessary

Reply:
that's needed because `_setup_git` is called in the `get_*` functions, and given communication works via stdout, we can't have popd/pushd's output here

Reply:
Aaah yes mybad